Modrinth/overrides

```diff
@@ -0,0 +1 @@
+/home/emma/repos/modrinth/own/fabulously-optimized/Curseforge/overrides
```

Comment:
This repo has no /home/emma ;)
See https://github.com/Madis0/fabulously-optimized/blob/main/MultiMC/Fabulously Optimized x.y.z/minecraft/config

Reply:
Ooooops haha, fixed in 99fd862